geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -1208,6 +1208,9 @@ private void initialize() {
           this.system.getConfig());
       initializeDeclarativeCache();
       completedCacheXml = true;
+    } catch (CacheXmlException e) {
```

Comment:
@dschneider-pivotal Thank you for your feedback. I've replaced CacheXmlException with RuntimeException. Sorry for two duplicate commits instead of the only one.